### PR TITLE
gha: Add cloud-hypervisor (runtime-rs) support to cri-containerd tests

### LINKED ISSUE #9181: gha: Add cloud-hypervisor (runtime-rs) support to cri-containerd tests

### PR DESCRIPTION
**Pull request by GabyCT**
This PR adds the Cloud Hypervisor driver, integrated with the runtime-rs, as part of the cri-containerd tests.

Fixes #9181

### CODE REVIEW

**Review by amshinde — APPROVED** [review]

File: tests/integration/cri-containerd/gha-run.sh [hunk 1]
```diff
@@ -48,7 +48,7 @@ function install_dependencies() {
 
 	for github_dep in "${github_deps[@]}"; do
 		IFS=":" read -r -a dep <<< "${github_dep}"
-		install_${dep[0]} "${dep[1]}"
+		install_"${dep[0]}" "${dep[1]}"
```
Comment (dborquez) [inline tests/integration/cri-containerd/gha-run.sh:51]: Here you could enclose the entire word using double quotes:
```suggestion
		"install_${dep[0]}" "${dep[1]}"
```

File: tests/integration/cri-containerd/gha-run.sh [hunk 2]
```diff
@@ -35,7 +35,7 @@ function install_dependencies() {
 	sudo apt-get -y install "${system_deps[@]}"
 
 	ensure_yq
-	${repo_root_dir}/tests/install_go.sh -p -f
+	"${repo_root_dir}"/tests/install_go.sh -p -f
```
Comment (dborquez) [inline tests/integration/cri-containerd/gha-run.sh:38]: You could enclose the path and the command using double quotes, and call it with the source command (which is an alias for `.` )
```suggestion
	. "${repo_root_dir}/tests/install_go.sh" -p -f
```

**Review by dborquez — CHANGES_REQUESTED** [review]
LGTM but I leave a few comments.

**Review by dborquez — APPROVED** [review]
LGTM thank you @GabyCT

**Pull request merged**